spec/clamby/command_spec.rb

```diff
@@ -75,5 +75,36 @@
         described_class.scan(good_path)
       end
     end
+
+    describe 'specifying config-file' do
+      it 'does not include the parameter in the clamscan command by default' do
+        Clamby.configure(daemonize: false, stream: false, fdpass: false)
```

Comment:
I had to explicitly set a lot of the config options here, I think due to leakage between tests - is this something that you've seen before?